packages/FlexGrid/__tests__/FlexGrid.spec.jsx

```diff
@@ -65,31 +64,22 @@ describe('FlexGrid', () => {
   })
 
   it('passes additional attributes to the element', () => {
-    const { flexGrid } = doMount({ id: 'the-id', 'data-some-attr': 'some value' })
+    const { flexGrid } = doMount({
+      id: 'the-id',
+      'data-some-attr': 'some value',
+    })
 
     expect(flexGrid).toHaveProp('id', 'the-id')
     expect(flexGrid).toHaveProp('data-some-attr', 'some value')
   })
 
   it('can have limited width at various viewport sizes', () => {
     mockMatchMedia(360)
-    const { flexGrid } = doMount({ limitWidth: true })
+    const { flexGrid } = doMount()
 
     expect(flexGrid).toHaveClassName('limitWidth')
   })
 
-  it('can be horizontally centred', () => {
-    const { flexGrid } = doMount({ limitWidth: true, centre: true })
-
-    expect(flexGrid).toHaveClassName('centre')
-  })
-
-  it('will show a centre is deprecated warning', () => {
-    doMount({ centre: true })
-
-    expect(deprecate).toHaveBeenCalled()
-  })
-
   it('does not allow custom CSS', () => {
```

Comment:
I think we should have a test for the case when `limitWidth={false}`. That code path is not unit tested currently.